.github/workflows/banana-ai-cicd.yml

```
@@ -21,18 +21,18 @@ jobs:
     timeout-minutes: 15
     steps:
       - uses: actions/checkout@v4  # Updated to v4 for security
-      
+
       - name: Set up Python
         uses: actions/setup-python@v4
```

actionlint (1.7.7)

the runner of "actions/setup-python@v4" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)